Demo-Mac/DemoAppController.m

```diff
@@ -302,6 +302,7 @@ - (void) updateSyncStatusView {
             tooltip = @"Syncing data...";
             break;
         default:
+            value = 0;
```

Comment:
This change is due to the fact that `value` was initialised when the `default` case was hit.